pyodbc.cpython-39-x86_64-linux-gnu.so,sha256=sSrMRHh0saUARyDFoHANLYK02Zx18z7H1sV0VgNtKLg,869208
pyodbc.pyi,sha256=YSHcS8DHzgggA5eKVJuBpvz6RAOSaO39BqTUEl7GA2c,32673
pyodbc-5.3.0.dist-info/METADATA,sha256=c_RU15asESBnr5pxQtvDKBD_QYsbx9sAIh-FxIVoF18,2744
pyodbc-5.3.0.dist-info/WHEEL,sha256=5iXtvL-Ee6MxEei7h3rrF-PfLYYJzI7hBjqY9zzRE3g,184
pyodbc-5.3.0.dist-info/top_level.txt,sha256=Rb69vS-SNN_Bfh99vX7eCcvEAJoyzowG3u4R7W36Flw,8
pyodbc-5.3.0.dist-info/RECORD,,
pyodbc-5.3.0.dist-info/licenses/LICENSE.txt,sha256=mFnIV3DJG3hmsPdpyTjjbWTQJgFBtSm5R7Tq2WiD6rQ,868
